### DISCUSSION TITLE
Gnome 42 with libadwaita theme

### DISCUSSION THREAD

**Discussion by sith-on-mars** [tsujan/Kvantum#779] [Ideas]
Would it be possible to have the libadwaita theme? Currently, Qt apps look out of place even with [adw-gtk3](https://github.com/lassekongo83/adw-gtk3#adw-gtk3).

**Comment by tsujan** [comment]
It's possible to have almost any kind of theme. But if you ask me (the author), I'm afraid I don't have the time.

**Comment by mgord9518** [comment]
@sith-on-mars It appears someone has been working on exactly [that](https://github.com/GabePoel/KvLibadwaita). Idk if it existed at the time of asking, but it does now
